experimental/PropertyDDS/examples/partial-checkout/package.json

```diff
@@ -12,11 +12,9 @@
   "author": "Microsoft and contributors",
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint",
-    "build:compile": "concurrently npm:tsc npm:build:webpack",
-    "build:full": "npm run build",
-    "build:full:compile": "npm run build:compile",
-    "build:webpack": "npm run webpack",
-    "build:webpack:dev": "webpack --env.clean",
+    "build:compile": "npm run tsc",
+    "build:full": "concurrently npm:build npm:webpack",
+    "build:full:compile": "concurrently npm:build:compile npm:webpack",
```

Comment:
You may need to make some changes to fluid-build since it might complain about scripts not being what it expects. Shouldn't block this though.

Reply:
It's happy as long as `build:webpack` is absent (this makes it the same set of scripts as most of our demos, e.g. diceroller)